script.js

```diff
@@ -1,4 +1,147 @@
+var gameState = 0;
+var diceCombinedPlayerOneFirst = [];
+var diceCombinedPlayerOneSecond = [];
+var diceCombinedPlayerTwoFirst = [];
+var diceCombinedPlayerTwoSecond = [];
+var playerOneFinalResult = "";
+var playerTwoFinalResult = "";
+
+//START OF THE GAME//
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  var output = document.querySelector("#submit-button");
```

Comment:
wowowo 👏 one advice here is that we can usually keep all our DOM selectors as global constans i.e. in global scope:
```
var SUBMIT_BUTTON = document.querySelector("#submit-button")
```
Super awesome that you picked this up though well done